net-p2p/tremc/tremc-0.9.1.ebuild

```diff
@@ -0,0 +1,38 @@
+# Copyright 1999-2020 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+PYTHON_COMPAT=( python3_{6,7} )
```

Comment:
Why no love for 3.8?

Reply:
`dev-lang/geoip-python` only has python3_{6,7}, it's my understanding that reverse dependencies are limited to the impls the dependency has.

Reply:
Could you test it? That is, if it doesn't turn out to require too many packages.

Reply:
Test geoip-python so both can have 3.8 or just tremc?

Reply:
In any case, geoip-python tests are successful on python3_{6,7,8}, I can open a new PR with 3.8 added, if you'd like.

Reply:
Both. Just add it as another commit to this PR.